### DISCUSSION TITLE
Tooltip on disabled button

### DISCUSSION THREAD

**Discussion by shyamajp** [twilio-labs/paste#1882] [Q&A] [answered]
Hello. We have some (anti-)patterns where disable buttons show the reason why it is being disabled on hover with tooltip as below. But `Tooltip` here never shows up on hover when `disabled` in `Button` is set to true. Is there any work around to force-show the tooltip as expected?
(In most of the code base, we are trying to replace with `Alert` for more visibility but this is just one of the few cases we have some old patterns left)

Thank you very much in advance.

```
<Tooltip text='this is never visible'>
    <Button variant='primary' disabled />
        Disabled Button
    </Button>
</Tooltip>
```

**Comment by andioneto** [comment, marked as answer]
Hi @shyamajp 👋 Due to accessibility concerns, there is not a work around to force-show the tooltip. You can learn more about why and potential alternatives by checking out [Simon's response in this discussion](https://github.com/twilio-labs/paste/discussions/765#discussioncomment-80009). 

If you want to dig deeper into the accessibility concerns, check out the links shared in [Nora's response here.](https://github.com/twilio-labs/paste/discussions/1247#discussioncomment-433977)

If you'd like to generally learn more about the differences between tooltip and popover (including when to use one or the other) please check out this [video Shadi made that outlines different use cases](https://paste.twilio.design/components/tooltip/#tooltip-compared-to-popover).
